packages/atomizer/src/rules.js

```diff
@@ -2867,6 +2867,71 @@ module.exports = [
     },
     /**
     ==================================================================
+    PLACE-CONTENT
+    ==================================================================
+    */
+    {
+        type: 'pattern',
+        name: 'Place content',
+        matcher: 'Pc',
+        allowParamToValue: false,
+        styles: {
+            'place-content': '$0 $1',
+        },
+        arguments: [
+            sortKeys({
```

Comment:
I think we can reuse... 1 sec..

Reply:
should this be https://www.w3.org/TR/css-align-3/#place-self-property `[<'align-self'>](https://www.w3.org/TR/css-align-3/#propdef-align-self) [<'justify-self'>](https://www.w3.org/TR/css-align-3/#propdef-justify-self)[?](https://www.w3.org/TR/css-values-4/#mult-opt)`